### PR TITLE
[CIAS30-3683] Restore SessionScheduleJob jobs

### PR DESCRIPTION
**Pull request by OskarDybowskiHTD**
## Related tasks
- [CIAS-3683](https://htdevelopers.atlassian.net/browse/CIAS30-3683)

## What's new?
-

### CODE REVIEW

**Review by WiktoriaRozanska — COMMENTED** [review]
Look at tests and merge dev into this branch

File: lib/tasks/one_time_use/recreate_session_schedule_jobs.rake
[the remark this answers is not in the document]
Comment (OskarDybowskiHTD) [reply lib/tasks/one_time_use/recreate_session_schedule_jobs.rake:1]: I am not sure if it's what you meant but done 👌

File: lib/tasks/one_time_use/recreate_session_schedule_jobs.rake [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+def user_sessions
+  UserSession.where(started: false).where.not(scheduled_at: nil)
+end
+
+def perform_due_schedule(user_session)
+  SessionScheduleJob.perform_now(
+    user_session.session.id,
+    user_session.user.id,
+    user_session.health_clinic,
+    user_session.user_intervention.id
```
Comment (WiktoriaRozanska) [inline lib/tasks/one_time_use/recreate_session_schedule_jobs.rake:36]: We need to pass `user_intervention_id` in `app/services/v1/user_session_schedule_service.rb:51`

**Review by WiktoriaRozanska — APPROVED** [review]

File: lib/tasks/one_time_use/recreate_session_schedule_jobs.rake (hunk 1 above)
Comment (OskarDybowskiHTD) [reply lib/tasks/one_time_use/recreate_session_schedule_jobs.rake:36]: I have already refactored this service in another task

**Pull request merged**